<ui version="4.0" >
 <class>Anmeldung</class>
 <widget class="QDialog" name="Anmeldung" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblVorname" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Vorname:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPasswort" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passwort:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWiederholen" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Passwort
wiederholen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNachname" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Nachname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtVorname" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>10</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtNachname" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>40</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPasswort" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>70</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtWiederholen" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>100</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>130</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbbrechen" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>130</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
